qabelbox/src/main/java/de/qabel/qabelbox/activities/BaseWizardActivity.java

```diff
@@ -115,7 +113,7 @@ public void onBackPressed() {
             if (canExit) {
                 finish();
             } else {
-
+                // TODO: This approach might be a bit shaky in some languages. Better store the whole msg string in xml instead of composing it here.
```

Comment:
pls explain this comment.. the strings are already loaded from xml. how do you want to compose the complete message with all buttons, headline in one entry?